tls/server/server.go

```diff
@@ -17,10 +17,10 @@ import (
 	"github.com/darvaza-proxy/darvaza/tls/sni"
 )
 
+// Proxy is a configuration for a TLSproxy
 type Proxy struct {
```

Comment:
I would feel better if this one was called `ProxyConfig`, and `proxy` renamed to `Proxy`

Reply:
fixed in f0a6318b5302e4a333d5a6761231049e80a19b77